scripts/jenkins/qat/Makefile

```diff
@@ -4,7 +4,7 @@
 #
```

Comment:
How this and below changes are related to FPGA?

Reply:
Refactoring of QAT related files is due to the fact that test were blocking each other. For instance, there is a QAT condition (see, [QAT condition](https://github.com/intel/intel-device-plugins-for-kubernetes/blob/master/scripts/jenkins/qat/plugin-deploy.sh#L17)) that never happens because the test-fpga-region pod was blocking it. Also, there is not only one state of the cluster that need to be saved, the state of the cluster is different when QAT and FPGA are deployed. Therefore, I came up with a refactoring that allows to test in bare metal specific hardware without affecting each other. First, a plugin is deployed (so far, there is QAT and FPGA but they may be others). Second, tests are run depending on the plugin. Third, any logs are captured from pods of interest. Fourth, the state of cluster is captured containing only the artifacts related to the specific plugin. And fifth, all the artifacts related to the plugin are removed so any subsequent plugin deployment is done over a healthy Kubernetes node. Maybe @mythi want to take a look to those changes on the QAT side. Notice that both folders, QAT and FPGA, follow the same structure and logic mentioned above.